samples/delete_deployment.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020, Google LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Delete a Game Servers Deployment.
+ * @param {string} projectId string project identifier
+ * @param {string} deploymentId unique identifier for the new Game Server Deployment
+ */
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  deploymentId = 'DEPLOYMENT_ID'
+) {
+  // [START cloud_game_servers_delete_deployment]
+  /**
```

Comment:
Nit, typically we add these blocks that have variables to uncomment after the client is instantiated.

Reply:
(here and below)

Reply:
Are you suggesting since they are input variables, it is redundant?

Reply:
Not at all!  Just asking to move lines 26->30 down to line 36.

Reply:
Done